paparazzi-gradle-plugin/src/test/projects/text-appearances-code/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <!-- Empty style for when you don't want a parent. -->
+  <style name="None" />
+
+  <style name="TextAppearance.Cash.Title" parent="None">
```

Comment:
Should this have a Cash name? or just a generic name?

Reply:
can replace